.travis.yml

```diff
@@ -12,6 +12,16 @@ matrix:
         - BUILD_APIDOC=true
         # Auth token to push API documentation
         - secure: "JKeXk8p65hodb12PVRST6A90swsNubc+46EbSJGSghldIxbFWLBAlwU+KLeOMO4V0veu6k4lnMa50V0UYFZmoUsS6W0aL5Ybo98SpzXHiNLOmOluoqJoF9TBsOTCCRFbWbccgJyVEtulgRcdml96naS51lq9Sw/VO/N3Z472304="
+    - os: linix
+      language: python
+      python: 2.7.14
+      install:
+        - pip install pytest
+      script:
+        - pytest
+      before_script: echo ""
+      before_cache: echo ""
+      env: NAME="Pytest"
```

Comment:
Please go with `BUILD_<stage name>` (here: `BUILD_PYTEST`) as environment variable to match the logic in the other build matrix selectors.

Reply:
This variable is used only for user interface of travis.  I plan to add same variable to compile stage too.
<img width="675" alt="name" src="https://user-images.githubusercontent.com/171597/35782636-d20c11e8-0a0b-11e8-99eb-43d6fc6a2596.png">